modules/assistant.js

```diff
@@ -104,12 +104,12 @@ MixIn(PrefCustomization, SimpleCustomization.prototype);
 
 
 function MultipleCustomization(aParams) {
-  this.customizations = [new PrefCustomization(p) for (p of aParams)];
+  this.customizations = !!aParams ? aParams.map(p => new PrefCustomization(p)) : [];
```

Comment:
That's a good point! By having a deeper look at the code, it seems like this is only used at modules/assistant.js:133, so it might be safe to just change that to `this.customizations = aParams.map(p => new PrefCustomization(p));`.

That approach is safer (but one could argue that it depends on how it's used) if the argument can be null or undefined, which doesn't seem the case here. My bad :-)